storage/types.go

```diff
@@ -33,19 +33,61 @@ type Node struct {
 }
 
 // NodeID uniquely identifies a Node within a versioned MerkleTree.
+// NodeIDs as presented to the storage layer will always have a multiple
```

Comment:
This "multiple of 8" comment is slightly misleading as it's a storage's restriction rather than NodeID's. Maybe put this comment after the non-multiple-of-8 example?

Reply:
Moved.